tests/unit/local/docker/test_lambda_container.py

```diff
@@ -544,17 +510,6 @@ def test_must_provide_container_env_vars(self, runtime):
 
         self.assertIsNotNone(container_env_vars)
 
-    @parameterized.expand([param(r) for r in set(RUNTIMES_WITH_ENTRYPOINT) if not r.startswith("dotnetcore")])
-    def test_debug_arg_must_be_split_by_spaces_and_appended_to_entrypoint(self, runtime):
-        """
-        Debug args list is appended starting at second position in the array
-        """
-        expected_debug_args = ["a=b", "c=d", "e=f"]
-        result, _ = LambdaContainer._get_debug_settings(runtime, self.debug_options)
-        actual = result[1:4]
-
-        self.assertEqual(actual, expected_debug_args)
-
```

Comment:
explain: `RUNTIMES_WITH_ENTRYPOINT` consists of dotnetcore and go1.x, and this test won't apply to go1.x anymore. So if I add `not r.startswith("go")` to the `@parameterized.expand`, this test won't run at all. So I removed it.